docs/dev-docs/conf.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+from datetime import date
+from sphinx.util import logging
+import recommonmark
+from recommonmark.transform import AutoStructify
+from sphinx_scylladb_theme.utils import multiversion_regex_builder
+
+logger = logging.getLogger(__name__)
+sys.path.insert(0, os.path.abspath('../..'))
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+needs_sphinx = '1.8'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.todo',
+    'sphinx.ext.mathjax',
+    'sphinx.ext.githubpages',
+    'sphinx.ext.extlinks',
+    'sphinx_scylladb_theme',
+    'sphinx_multiversion',
+    'recommonmark',
+    'sphinx_markdown_tables',
+]
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+source_suffix = ['.rst', '.md']
+autosectionlabel_prefix_document = True
+
+# The encoding of source files.
+#
+# source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'contents'
```

Comment:
If you only want to build docs for the manual folder, consider:

```suggestion
master_doc = 'manuals/getting-started/index.rst'
```